tinytodo/src/timebox.rs

```diff
@@ -0,0 +1,192 @@
+use std::time::{Duration, SystemTime, UNIX_EPOCH};
+
+use cedar_policy::{Policy, PolicyId};
+use itertools::Either;
+use serde_json::json;
+use tracing::info;
+
+use crate::{
+    context,
+    entitystore::EntityStore,
+    objects::Timebox,
+    policy_store::PolicyStore,
+    util::{EntityUid, ListUid, TimeBoxUid, UserOrTeamUid, TYPE_TIMEBOX, TYPE_USER},
+};
+
+#[tracing::instrument]
+pub fn update_timebox(
```

Comment:
[nit] It might be better to define a struct to pass in as the argument instead of passing in 5 arguments.